control_panel_api/settings/base.py

```diff
@@ -142,6 +142,9 @@
 OIDC_CLIENT_ID = os.environ.get('OIDC_CLIENT_ID')
 OIDC_CLIENT_SECRET = os.environ.get('OIDC_CLIENT_SECRET')
 OIDC_DOMAIN = os.environ.get('OIDC_DOMAIN')
+OIDC_FIELD_USERNAME = 'nickname'
+OIDC_FIELD_EMAIL = 'email'
+OIDC_FIELD_NAME = 'name'
```

Comment:
nice 👍

Reply:
These aren't standard field names in OIDC - they're from Github.